crates/pixi_config/src/lib.rs

```diff
@@ -1824,7 +1861,8 @@ pub fn config_path_system() -> PathBuf {
 /// Returns the path(s) to the global pixi config file.
 pub fn config_path_global() -> Vec<PathBuf> {
     vec![
-        // On macos, add the XDG_CONFIG_HOME directory as well, although it's not a standard and not set by default.
+        // On macos, add the XDG_CONFIG_HOME directory as well, although it's not a standard and
+        // not set by default.
```

Comment:
I think we should use this crate https://lib.rs/crates/dirs to easily get correct directories.

Reply:
I didnt change this, only retouched the comments. But I agree.

Reply:
Could you quickly try to fix that? But it'll take more time than expected leave it like this until better times :)

Reply:
While looking into it it seems we already do! `dirs::config_dir` is used a few lines below. The comment explicitly states why `XDG_CONFIG_HOME` is additionally used on macos.